packages/core/pluggableElementTypes/renderers/ServerSideRendererType.ts

```diff
@@ -168,11 +168,6 @@ export default class ServerSideRenderer extends RendererType {
     return deserialized
   }
 
-  // will expand if soft clipping or feature glyphs are shown
-  getExpandedRegion(region: Region, renderArgs: RenderArgsDeserialized) {
-    return region
-  }
-
```

Comment:
@peterkxie It didn't look like this was doing anything, so I removed it. Do you know if there's a reason to keep it?

Reply:
this function is used by derived subclasses PileupRenderer, BoxRenderer, might not wanna remove?

Reply:
what's the update on this?

Reply:
I restored it